test/include.js

```diff
@@ -57,13 +54,13 @@ test('loads a thrift file that imports synchronously', function t(assert) {
     );
 
     assert.deepEqual(
-        mainThrift.consts.nums.value,
+        mainThrift.consts.nums,
```

Comment:
This is also a backward incompatible change. Constants were previously exposed as the model object, not their surface value.